docs/get-started/Work-with-pyspark.ipynb

```diff
@@ -0,0 +1,145 @@
+      "24/04/03 22:05:00 WARN Client: Neither spark.yarn.jars nor spark.yarn.archive is set, falling back to uploading libraries under SPARK_HOME.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import findspark\n",
+    "findspark.init()\n",
+    "\n",
+    "from pyspark import SparkConf, SparkContext\n",
+    "nativesql_jars = \"/path/to/gluten-XXXX.jar\"\n",
+    "conf = SparkConf().setAppName(\"PySpark Gluten\").setMaster(\"yarn\")\n",
+    "conf.set(\"spark.executor.instances\", \"2\")\n",
+    "conf.set(\"spark.executor.memory\", \"6g\")\n",
+    "conf.set(\"spark.executor.cores\", \"2\")\n",
+    "conf.set(\"spark.driver.memory\", \"2g\")\n",
+    "conf.set(\"spark.memory.offHeap.enabled\", \"true\")\n",
+    "conf.set(\"spark.memory.offHeap.size\", \"2g\")\n",
+    "conf.set(\"spark.executor.memoryOverhead\", \"384M\")\n",
+    "conf.set(\"spark.driver.extraClassPath\", nativesql_jars)\n",
+    "conf.set(\"spark.executor.extraClassPath\", nativesql_jars)\n",
+    "conf.set(\"spark.plugins\", \"org.apache.gluten.GlutenPlugin\")\n",
+    "conf.set(\"spark.gluten.sql.columnar.backend.lib\", \"velox\")\n",
+    "conf.set(\"spark.gluten.loadLibFromJar\", \"false\")\n",
+    "conf.set(\"spark.shuffle.manager\", \"org.apache.spark.shuffle.sort.ColumnarShuffleManager\")\n",
+    "sc = SparkContext(conf=conf)\n",
+    "from pyspark.sql import SparkSession\n",
+    "spark_session = SparkSession(sc)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6c4ec66a-3b8c-4a65-b948-8420b492333d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "== Physical Plan ==\n",
+      "VeloxColumnarToRowExec\n",
+      "+- ^(1) FilterExecTransformer (isnotnull(category#1) AND (category#1 = Cellphone))\n",
+      "   +- ^(1) InputIteratorTransformer[product#0, category#1, revenue#2L]\n",
+      "      +- ^(1) InputAdapter\n",
+      "         +- ^(1) RowToVeloxColumnar\n",
+      "            +- *(1) Scan ExistingRDD[product#0,category#1,revenue#2L]\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "df = spark_session.createDataFrame(\n",
+    "    [\n",
+    "        (\"Normal\", \"Cellphone\", 6000),\n",
+    "        (\"Normal\", \"Tablet\", 1500),\n",
+    "        (\"Mini\", \"Tablet\", 5500),\n",
+    "        (\"Mini\", \"Cellphone\", 5000),\n",
+    "        (\"Foldable\", \"Cellphone\", 6500),\n",
+    "        (\"Foldable\", \"Tablet\", 2500),\n",
+    "        (\"Pro\", \"Cellphone\", 3000),\n",
+    "        (\"Pro\", \"Tablet\", 4000),\n",
+    "        (\"Pro Max\", \"Cellphone\", 4500)\n",
+    "    ],\n",
+    "    [\"product\", \"category\", \"revenue\"]\n",
+    ")\n",
+    "df.filter(\"category = 'Cellphone'\").explain()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47e7593d-aaf7-4608-b13a-f0d948f39a55",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```

Comment:
This one rendered as an empty block. Is it intended?

https://github.com/apache/incubator-gluten/blob/36bff654a017a19c2a3acd9223f71d0b330bdd41/docs/get-started/Work-with-pyspark.ipynb

Reply:
Removed.